src/functionaltests/java/com/ericsson/ei/subscriptions/trigger/SubscriptionTriggerSteps.java

```diff
@@ -294,10 +286,19 @@ private void setupRestEndpoints() {
      * @throws IOException
      */
     private void setupSMTPServer() throws IOException {
-        int port = SocketUtils.findAvailableTcpPort();
-        LOGGER.debug("Setting SMTP port to " + port);
-        mailSender.setPort(port);
-        smtpServer = SimpleSmtpServer.start(port);
+        boolean connected = false;
+        while (!connected) {
+            try {
+                int port = SocketUtils.findAvailableTcpPort();
+                LOGGER.debug("Setting SMTP port to " + port);
+                mailSender.setPort(port);
+                smtpServer = SimpleSmtpServer.start(port);
+                // connected, go on
+                connected = true;
+            } catch (BindException e) {
+                // address already in use. Try again!
```

Comment:
Maybe write something here if it goes wrong?

Reply:
Done